software/wakupator.yml

```
@@ -0,0 +1,10 @@
+name: "Wakupator"
+website_url: "https://github.com/Gibus21250/Wakupator"
+source_code_url: "https://github.com/Gibus21250/Wakupator"
+description: "Wake On LAN Machine Manager based on network traffic."
+licenses:
+  - MIT
+platforms:
+  - C
+tags:
+  - Automation
```

Automation is slightly out of the scope for this project. There are other wake on lan projects in misc.
```suggestion
  - Miscellaneous
```

---

Based on the definition provided in the file automation.yml, "software designed to reduce human intervention in processes."

As Wakupator automates the start-up of machines when traffic is detected, it removes the need for a human to start them up manually.

That's why I have tagged Wakupator as "automation". It's not an interface to WoL machines, but an automatic software which start machines using Wake On Lan.

---

I think this would be more suited to https://github.com/awesome-foss/awesome-sysadmin